src/components/SearchForm/SearchForm.stories.js

```diff
@@ -0,0 +1,22 @@
+import SearchForm from "./SearchForm";
+import './SearchForm.scss';
+
+export default {
+    title: 'components/SearchForm',
+    component: SearchForm,
+    tags: ['autodocs'],
+    argTypes: {
+        initialSearchQuery: { control: 'text' }
+    },
+}
+
+export const NoInitialQuery = () => {
+    return <SearchForm/>
```

Comment:
You will see error after form submission. You need to pass args anyway, because storybook will read component's prop types and it automatically append it.